hbase-common/src/main/java/org/apache/omid/transaction/CellUtils.java

```diff
@@ -52,6 +52,7 @@
     static byte[] DELETE_TOMBSTONE = Bytes.toBytes("__OMID_TOMBSTONE__");
```

Comment:
Will Cells end up with these constant values and if so can we make them shorter?